### PR TITLE
Update README.md to include credentials info

### PR DESCRIPTION
**Pull request by wilsonw926**
Update README.md to include credentials info

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,6 +23,26 @@ rails db:setup
 # This runs db:create, db:schema:load, and db:seed
```
Comment (FranCogMents) [inline README.md]: what version of postgresql? (let's make sure this version that is listed on readme is the same that is used for production on heroku)

Comment (wilsonw926) [reply README.md]: 👍

Comment (FranCogMents) [reply README.md]: heh i think there might be an added decimal

Comment (wilsonw926) [reply README.md]: cool good catch.

**Pull request merged**